api/src/main/java/org/apache/iceberg/TableScan.java

```diff
@@ -69,7 +69,10 @@ default TableScan select(String... columns) {
    * @param toSnapshotId read append data up to this snapshot id
    * @return a table scan which can read append data from {@code fromSnapshotId} exclusive and up to
    *     {@code toSnapshotId} inclusive
+   * @deprecated since 1.0.0, will be removed in 1.1.0; use {@link Table#newIncrementalAppendScan()}
```

Comment:
I assume we decided that the next major/minor release will be 1.0.0. there won't be a 0.15.0.

Reply:
Yeah, that seems to be the assumption. @rdblue, could you confirm?

Reply:
Yes, this should be removed in 2.0.0.

Reply:
Fixed.